### PR TITLE
include a different name tag value.

### PR DESCRIPTION
**Pull request by xjaco222x**
include a different name tag value.

The tag value can have special characters that an image name does not accept (ex | /)

### CODE REVIEW

File: Base2DBarcode.php [hunk 1]
```diff
@@ -318,8 +318,11 @@ public function getBarcodeFilenameFromGenPath($path)
      *
      * @return bool
      */
-    public function getBarcodePNGPath($code, $type, $w=3, $h=3, $color=array(0, 0, 0))
+    public function getBarcodePNGPath($code, $type, $name, $w=3, $h=3, $color=array(0, 0, 0))
     {
+        // set name of image png
+        $name = $name;
```
Comment (paterik) [inline Base2DBarcode.php]: not necessary to set $name = $name

**Comment by xjaco222x** [comment]
It was just to make an easy understanding of the more layman User.

**Comment by xjaco222x** [comment]
correction in name inclusion on the label 1d.

**Comment by paterik** [comment]
thanx :)

**Pull request merged**